Add ShowProgressAsPercentage option

danieroux opened the pull request:
If progress is show: This toggles between showing the percentage completed and the number of pages completed.

Also added a battery unicode symbol in front of the battery display, to clearly show the point of that percentage.

geometer commented:
Good update, thanks. 2 questions:

1) That could be confusing for users, to have both page number & page percentage checkboxes in the preferences, nope? May be it makes sense to create option for 'position displaying format', with options
* do not display
* show (approximate) page number / total page number
* show percentage
* (?) show both

2) What will happen if there is no battery symbol in the font? (This symbol is great idea, thanks, but this question...)

danieroux commented:
I've changed it to a list of options, like you suggested. This does have a translation impact, so I've removed the old resource key.

Most fonts will have the symbol. If it doesn't, it should render as a rectangle, meaning "missing glyph". Which would still sort of look like a battery!

danieroux commented:
Should I make this change on the 2.6 branch instead?

geometer commented:
Sorry, I'm currently on holidays till 19th September. Still working some evenings, but not too much. I remember about your patch, and plan to merge it (still not sure about battery symbol). It's not necessary to change something for 2.6.

geometer commented:
Here is how battery symbol looks on Galaxy Tab:
![battery_symbol](https://cloud.githubusercontent.com/assets/344328/10086485/8e1ec420-6306-11e5-958e-5280eec22822.png)
